tests/cypress/integration/issue_1498_message_ui_raw_labels_wrong.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Message in UI when raw labels are wrong.', () => {
+
+    const issueId = '1498'
+    const labelName = `Issue ${issueId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    let taskRaw = [
+        {
+          name: labelName,
+          id: 1,
+          color: '#c4a71f',
+          attributes: [
+            {
+              id: 1,
+              name: attrName,
+              input_type: 'text',
+              mutable: false,
+              values: [
+                textDefaultValue
+              ]
+            }
+          ]
+        }
+      ]
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.contains('button', 'Create new task').click()
```

Comment:
Let's use id ``cvat-create-task-button`` to get the element

Reply:
Done. Also changed it in the "createAnnotationTask" command.